app/models/sidebar_section_link.rb

```diff
@@ -37,6 +43,7 @@ class SidebarSectionLink < ActiveRecord::Base
 #  created_at         :datetime         not null
 #  updated_at         :datetime         not null
 #  sidebar_section_id :integer
+#  position           :integer          default(0), not null
```

Comment:
I'm wondering if an index on this column with `sidebar_section_id` and `user_id` will help?